Stratis.Bitcoin/Features/Consensus/TimeSyncBehavior.cs

```diff
@@ -44,10 +44,10 @@ public struct TimestampOffsetSample
         }
 
         /// <summary>Maximal number of samples to keep inside <see cref="inboundTimestampOffsets"/>.</summary>
-        private const int MaxInboundSamples = 200;
+        public const int MaxInboundSamples = 200;
```

Comment:
This is fine, are you are aware of the InternalVisibleTo attribute right?

Reply:
yes, found it ugly and never used :-)
but maybe i should reconsider in the future